stencil.config.ts

```diff
@@ -8,21 +8,27 @@ export const config: Config = {
   outputTargets: [
     {
       type: 'dist-custom-elements',
-      autoDefineCustomElements: true,
+      autoDefineCustomElements: true
     },
     {
       type: 'dist',
       esmLoaderPath: '../loader',
+      copy: [
+        {
+          src: 'components/*/*.scss',
+          dest: 'components-sass-files'
```

Comment:
after testing npm install `@laws-africa/web-components`, in a project and looking at node-modules, I saw that the src directory was not included in the package. So my strategy is to include all the component sass files in the dist directory.
So in the case of tausi, we have access to `structure.scss`

Reply:
Ok. Ionic has a `css` directory alongside their `dist` directory. That lets them distribute CSS separately from the components JS. Could we do something similar, except with the scss files?

![image](https://user-images.githubusercontent.com/4178542/145529831-c72b89f5-6c10-43ec-a62e-553a630f04ef.png)

So could we add a new `scss` directory here?

![image](https://user-images.githubusercontent.com/4178542/145529887-4bb3be4f-0755-483b-9bf1-412896b2ed9a.png)

Reply:
Or, if copying into this directory instead of `dist` is too difficult, just change the dir name to `scss`?

Reply:
done